CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -34,13 +35,22 @@
     @Inject
     public PoseSubsystem(WPIFactory factory, PropertyManager propManager) {
         log.info("Creating PoseSubsystem");
-        imu = factory.getGyro();
-        leftDistanceSensor = factory.getAnalogDistanceSensor(1, voltage -> VoltageMaps.placeholder(voltage));
+        imu = factory.getGyro(ImuType.navX);
+        leftDistanceSensor = factory.getAnalogDistanceSensor(1, voltage -> TemporaryVoltageMap.placeholder(voltage));
         leftSensorMountingDistanceInches = propManager.createPersistentProperty("LeftSensorMountingDistanceInches", 16.0);
-        
+        currentHeadingProp = propManager.createEphemeralProperty("CurrentHeading", 0.0);
```

Comment:
How do we want to account for different initial headings? That's always been a challenge.